web/src/ChooseArea.svelte

```diff
@@ -53,18 +62,27 @@
       local places are properly connected without severance.
     </p>
 
-    <h3>Select an area</h3>
+    <h3><label class="ds_label" for="component">Select an area</label></h3>
 
     <p>Select your area from this list, or click on the map, to start.</p>
 
-    <select bind:value={lad}>
-      <option value=""></option>
-      {#each ladNames as value}
-        <option {value}>{value}</option>
-      {/each}
-    </select>
+    <div class="ds_select-wrapper">
```

Comment:
In larger projects, the extra boilerplate required by gov.uk or SG or similar gets very unpleasant. For ATE projects, we made https://github.com/acteng/govuk-svelte to lessen this burden, but then we have kind of cumbersome, opinionated wrappers around normal HTML elements. For NPW for now, let's stick with this more verbose, explicit style, and revisit later.

Reply:
It's basically not too bad, I found. There's a bit more wrapping but we aren't dealing with thousands of controls. I would just go with the manual approach for now and we can wrap later if really needed.

Personally I think had they designed it now they probably would have managed to avoid a lot of the excess stuff as CSS has more control available now. I'm a huge advocate for retaining very purist HTML structures as that is best long-tem.